### PR TITLE
[Improvement-14371][Hivecli] Add support for hivevar setting in hivecli task

### LINKED ISSUE #14371: [Improvement][Hivecli] Add support for hivevar setting in hivecli task
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Add support for hivevar setting in hivecli

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by rickchengx**
## Purpose of the pull request

* close: #14371 

## Brief change log

use `hive -f xxx.sql` to submit hive task to avoid parse exceptions when using `set hivevar:xxx=xxx` 

截屏2023-06-28 17 32 04

## Verify this pull request
UT

### CODE REVIEW

**Comment by rickchengx** [comment]
Hi, @EricGao888 , could u please help review this

File: dolphinscheduler-task-plugin/dolphinscheduler-task-hivecli/src/main/java/org/apache/dolphinscheduler/plugin/task/hivecli/HiveCliTask.java [hunk 1]
```diff
@@ -108,29 +121,40 @@ protected String buildCommand() {
...
             }
 
-            args.add(StringUtils.stripStart(resourceInfos.get(0).getResourceName(), "/"));
+            try {
+                resourceFileName = resourceInfos.get(0).getResourceName();
+                sqlContent = FileUtils.readFileToString(
+                        new File(String.format("%s/%s", taskExecutionContext.getExecutePath(), resourceFileName)),
+                        StandardCharsets.UTF_8);
+            } catch (IOException e) {
+                log.error("read hive sql content from file {} error ", resourceFileName, e);
```
Comment (Radeity) [inline dolphinscheduler-task-plugin/dolphinscheduler-task-hivecli/src/main/java/org/apache/dolphinscheduler/plugin/task/hivecli/HiveCliTask.java:139]: Maybe we should throw exception here.

Comment (rickchengx) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-hivecli/src/main/java/org/apache/dolphinscheduler/plugin/task/hivecli/HiveCliTask.java:139]: Sure

**Comment by rickchengx** [comment]
Hi, @Radeity could u please review this again

**Review by SbloodyS — APPROVED** [review]
LGTM

**Comment by Radeity** [comment]
> Hi, @Radeity could u please review this again

A late +1 :D

**Pull request merged**